release/models/bfd/openconfig-bfd.yang

```diff
@@ -138,48 +144,61 @@ module openconfig-bfd {
           "The BFD echo function failed - echo packets have not been
           received for the required period of time.";
       }
-      enum FORWARDING_RESET {
+      enum NEIGHBOR_SESSION_DOWN {
         value 3;
+        description
+          "The neighbor has signaled the session down.";
+      }
+      enum FORWARDING_RESET {
+        value 4;
         description
           "The forwarding plane in the local system was reset - such
           that the remote system cannot rely on the forwarding state of
           the device specifying this error code.";
       }
       enum PATH_DOWN {
-        value 4;
```

Comment:
Per the other change, do these values correspond to the spec? Is it better for us to just remove them all?

Reply:
These values do now correspond to the spec but yes - we could just remove them all and state the design pattern is to never assign variant values but pay very close attention to ordering if auto assigned values do infact have side effects

https://datatracker.ietf.org/doc/html/rfc5880#section-4.1
https://datatracker.ietf.org/doc/html/rfc6428#section-5